config/initializers/dfe_analytics.rb

```diff
@@ -45,7 +45,7 @@
   # enable analytics. You might want to hook this up to a feature flag or
   # environment variable.
   #
-  config.enable_analytics = proc { FeatureFlag.active?(:dfe_analytics) }
+  config.enable_analytics = proc { !Rails.env.review? && FeatureFlag.active?(:dfe_analytics) }
```

Comment:
Could we not hardcode the environment name and use an environment variable instead? Could be set [here](https://github.com/DFE-Digital/early-careers-framework/blob/main/terraform/application/application.tf#L18) depending on the environment

Reply:
I'd be happy to do that but think having a feature flag **and** a env var is too much.

I'll add a backlog ticket to make it just use an env var.

Reply:
No worries